### ISSUE TITLE
Finetuning and model weights of InceptionV3

### ISSUE THREAD

**Issue by dingw00** [jihyechoi77/concepts-for-ood#2]
Hello,

Thank you for your impressive work. We are trying to reproduce the concept-based OOD method, however, I have some trouble finetuning the InceptionV3 model. I run the code `helper.load_model_inception_new(data_loader, pretrain=False)`, training the InceptionV3 model (pretrained on ImageNet1k) on AwA2 dataset for 20 epochs, with batch_size=256, Adam optimizer learning_rate=1e-4. The pretrained model has 537,394 trainable parameters, as shown in the screenshot: 
![Screenshot from 2024-07-18 17-31-38](https://github.com/user-attachments/assets/c692dba3-4dc0-4d26-b793-5ecd70b164f3)

However, I couldn't achieve the model accuracy of 92.13% as indicated in your paper. Even after training 30 epochs, I got
`117/117 ━━━━━━━━━━━━━━━━━━━━ 1022s 8s/step - accuracy: 0.7997 - loss: 1.2730 - val_accuracy: 0.8760 - val_loss: 0.9546`. The training accuracy is much smaller than the validation accuracy.

Could you please provide more details about how you finetuned the model? Otherwise could you provide the trained weights of InceptionV3? Thank you in advance!
